README.md

```diff
@@ -123,6 +123,27 @@ use the `--distro <distro>:<version>` flag. A full example is:
 grype --add-cpes-if-none --distro alpine:3.10 sbom:some-apline-3.10.spdx.json
 ```
 
+### Scan attestations
+Grype can scan SBOMs from attestations, as long as they are encoded as part of [in-toto envelope](https://github.com/in-toto/attestation/blob/main/spec/README.md#envelope).
```

Comment:
nit: I think we remove comma here

Reply:
Apologies if this came from my earlier suggestion

Reply:
Comment from meeting - Add information to feature session for this PR

Reply:
Comment from meeting - update scheme section to talk about new source and how it is a more secure version of the grype workflow

Reply:
Comment from meeting - update Scan attestations section Examples with full connective story of commands